webapp/store/page/dataExport/reducer.ts

```diff
@@ -16,7 +14,7 @@ const initialState: DataExportState = {
 }
 
 const actionHandlers = {
-  [AppActionTypes.appCountryIsoUpdate]: () => ({ ...initialState }),
+  'app/updateCountryIso': () => ({ ...initialState }),
```

Comment:
can we avoid hardcoded strings?

Reply:
In dataExport refactor will be handled, app throws error if `<action>.type` is used